API/testrunner/testrunner.py

```diff
@@ -30,6 +37,40 @@ def __init__(self, os):
         self.os = os
         self.results = []
 
+    def __update_status_icon(self):
+        '''
+        Update the status icon.
+        '''
+        app = self.os.get_app()
+
+        results_web_path = TEST_RESULTS_WEB_PATH[app.get_name()]
+        if not utils.exists(results_web_path):
+            return
+
+        status = 'passing'
+        for test in self.results:
+            if test['result'] == 'fail':
+                status = 'failing'
+                break
+
+        device = app.get_device()
+        current_status_icon = utils.join(results_web_path, 'status', '%s.svg' % device.get_type())
+
+        if utils.exists(current_status_icon):
+            return
+
+        with open(current_status_icon) as file:
+            if status in file.read():
+                return
+
+        image = 'pass.svg' if status is 'passing' else 'fail.svg'
+        copied_status_icon = utils.join(results_web_path, 'img', image)
+        utils.copy_file(copied_status_icon, current_status_icon)
+
+        utils.execute(results_web_path, 'git', ['add', current_status_icon])
```

Comment:
Are you sure we need to add the `current_status_icon` and not the `copied_status_icon`?

Reply:
Yes, because the `utils.copy_file` method copies the `copied_status_icon` into the path of the  `current_status_icon`.